packages/app/src/ChartUtils.tsx

```diff
@@ -172,7 +176,7 @@ export function seriesToSearchQuery({
           aggFn !== 'count' && field ? ` ${field}:*` : ''
         }${
           'groupBy' in s && s.groupBy != null && s.groupBy.length > 0
-            ? ` ${s.groupBy}:${groupByValue}`
+            ? ` ${s.groupBy}:${groupByValue ?? '*'}`
```

Comment:
groupBy * ?

Reply:
This fixes a bug where if the groupByValue is undefined (any group), we use the `group:*` operator correctly, previously we'd incorrectly search for `group:undefined`